docs/user-guide/troubleshooting.rst

```diff
@@ -26,6 +26,11 @@ CRITICAL, ERROR, WARNING, INFO, DEBUG, and NOTSET.
    rsyslog typically only displays INFO and higher, so if you set Pulp to DEBUG it will still be
    filtered by rsyslog. See the :ref:`rsyslogd` section for more information.
 
+.. note::
```

Comment:
Added a separate note to have separated context from the previous note. Let me know if it should be all inside the same note box.